File: tasks/translation/output/Qwen/Qwen2.5-32B-Instruct-AWQ/codenet/direct/inline/uncomment/direct/Java/Go/s063482767_commented.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	scanner := bufio.NewScanner(os.Stdin)
	scanner.Scan()
	n, _ := strconv.Atoi(scanner.Text())
	var queue []int
	for i := 0; i < n; i++ {
		scanner.Scan()
		num, _ := strconv.Atoi(scanner.Text())
		queue = append(queue, num)
	}
	sort.Ints(queue)
	sum := 0
	for i := 0; i < n-1; i++ {
		sum += queue[i]
	}

	if queue[n-1] < sum {
		fmt.Println("Yes")
	} else {
		fmt.Println("No")
	}
}
// End of Code

File: tasks/translation/output/Qwen/Qwen2.5-32B-Instruct-AWQ/codenet/direct/inline/uncomment/direct/Java/Python/s063482767_commented.py
import heapq

if __name__ == "__main__":
    # write your code here
    n = int(input())
    queue = []
    for _ in range(n):
        heapq.heappush(queue, int(input()))
    
    sum = 0
    for _ in range(n - 1):
        sum += heapq.heappop(queue)
    
    # 出力
    print("Yes" if heapq.heappop(queue) < sum else "No")  # End of Code
